blockchain/block/systemlog.go

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2020 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package block
+
+import (
+	"math/big"
+
+	"github.com/golang/protobuf/proto"
+
+	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/iotexproject/iotex-address/address"
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+
+	"github.com/iotexproject/iotex-core/action"
+	"github.com/iotexproject/iotex-core/pkg/util/byteutil"
+)
+
+// system log definitions
+type (
+	// TokenTxRecord is a token transaction record
+	TokenTxRecord struct {
+		amount    string
+		sender    string
+		recipient string
+	}
```

Comment:
shall we add a topic here to distinguish evm transfer, bucket withdraw and etc.

Reply:
good suggestion